# This controller is found in the joint_state_controller package and publishes
# the state of the robot's joints into ROS directly from Gazebo.
# The joint state controller handles publishing transforms for any moving joints
five_dof_robotarm:
 joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

 base_to_turntable_controller:
  type: effort_controllers/JointPositionController
  joint: base_to_turntable
  pid: {p: 30.0, i: 0.1, d: 10.0}

 second_joint_controller:
   type: effort_controllers/JointPositionController
   joint: second_motor_to_first_mounting_plate
   pid: {p: 200.0, i: 1, d: 10.0}

 third_joint_controller:
   type: effort_controllers/JointPositionController
   joint: third_motor_to_second_mounting_plate
   pid: {p: 50.0, i: 1, d: 10.0}

 fourth_joint_controller:
   type: effort_controllers/JointPositionController
   joint: fourth_motor_to_third_mounting_plate
   pid: {p: 10.0, i: 1, d: 10.0}
